WordPress/Classes/ViewRelated/Blog/BlogDetailsViewController.m

```diff
@@ -600,7 +600,8 @@ - (UITableViewScrollPosition)optimumScrollPositionForIndexPath:(NSIndexPath *)in
 - (void)configureTableViewData
 {
     NSMutableArray *marr = [NSMutableArray array];
-    if ([Feature enabled:FeatureFlagDomainCredit]) {
+    if ([DomainCreditEligibilityChecker canRedeemDomainCreditWithBlog:self.blog
```

Comment:
I would probably keep the previous if statement without using `isFeatureFlagOn` as argument. Only because when we will remove the flag it won't affect any change on the method signature

Reply:
yup it would be easier to remove the feature flag, and reads more clear at caller level - just updated in https://github.com/wordpress-mobile/WordPress-iOS/pull/11590/commits/93bf0111252a61cae5967024196203fef6817d31
also tested with flag off that the domain credit section is not shown